docs/source/savant_101/30_dm.rst

```diff
@@ -99,7 +99,10 @@ The parameter ``layer_names`` defines the names of the model output layers retur
 Converter
 ^^^^^^^^^
 
-``Output`` defines an important parameter ``converter`` which is basically a method which makes bounding boxes from a raw tensor. For "standard" detection models supported by DeepStream ``converter`` parameter is not required, however if the model's output cannot be parsed automatically, you have to provide an implementation of :py:class:`~savant.base.converter.BaseObjectModelOutputConverter` to produce boxes for detected objects.
+``Output`` defines an important parameter ``converter`` which is basically a method which makes bounding boxes from a raw tensor. 
+For detection models prepared for Nvidia DeepStream, the ``converter`` parameter is not required, however if the model's output cannot 
```

Comment:
Or use a custom lib and C converter, as DS suggests (DeepStream Yolo does it this way).